config/config.h

```diff
@@ -47,7 +47,9 @@
 
 /* Define to 1 if you have the declaration of `program_invocation_short_name',
    and to 0 if you don't. */
+#ifndef HAVE_DECL_PROGRAM_INVOCATION_SHORT_NAME
```

Comment:
I don't see similar ifndef stuff for other knobs that could be configured - I'm not very familiar with this part of the dispatch build system but this feels like it might not be the place to add this.

cc: @compnerd to weigh in.

Reply:
You're right. In fact, when `_GNU_SOURCE` is defined (on a level above, in the Swift build system) related errors go away and I don't have to add an `#ifndef` here. That's cleaned up now.